test/utils/queries/users-data.ts

```diff
@@ -1,283 +1,98 @@
+import { PlatformRole } from '@alkemio/client-lib';
 import { getUserDataCodegen } from '@test/functional-api/user-management/user.request.params';
 import {
-  assignPlatformRoleToUser,
-  assignUserAsGlobalCommunityAdmin,
   assignUserAsSupport,
+  assignUserAsGlobalCommunityAdmin,
+  assignPlatformRoleToUser,
 } from '../mutations/authorization-mutation';
 import { TestUser } from '../token.helper';
-import { PlatformRole } from '@alkemio/client-lib';
 
-let user: any;
+interface UserData {
+  email: string;
+  id: string;
+  displayName: string;
+  profileId: string;
+  nameId: string;
+  agentId: string;
+}
+
+interface Users {
+  globalAdmin: UserData;
+  globalCommunityAdmin: UserData;
+  globalSpacesAdmin: UserData;
+  spaceAdmin: UserData;
+  spaceMember: UserData;
+  challengeAdmin: UserData;
+  challengeMember: UserData;
+  opportunityAdmin: UserData;
+  opportunityMember: UserData;
+  qaUser: UserData;
+  notificationsAdmin: UserData;
+  nonSpaceMember: UserData;
+  betaTester: UserData;
+}
+
+const createUserData = (email: string): UserData => ({
+  email,
+  id: '',
+  displayName: '',
+  profileId: '',
+  nameId: '',
+  agentId: '',
+});
 
-export const usersSetEmail = [
-  {
-    email: 'admin@alkem.io',
-    id: '',
-    displayName: '',
-    profileId: '',
-    nameID: '',
-  },
-  {
-    email: 'community.admin@alkem.io',
-    id: '',
-    displayName: '',
-    profileId: '',
-    nameID: '',
-  },
-  {
-    email: 'global.spaces@alkem.io',
-    id: '',
-    displayName: '',
-    profileId: '',
-    nameID: '',
-  },
-  {
-    email: 'space.admin@alkem.io',
-    id: '',
-    displayName: '',
-    profileId: '',
-    nameID: '',
-  },
-  {
-    email: 'space.member@alkem.io',
-    id: '',
-    displayName: '',
-    profileId: '',
-    nameID: '',
-  },
-  {
-    email: 'challenge.admin@alkem.io',
-    id: '',
-    displayName: '',
-    profileId: '',
-    nameID: '',
-  },
-  {
-    email: 'challenge.member@alkem.io',
-    id: '',
-    displayName: '',
-    profileId: '',
-    nameID: '',
-  },
-  {
-    email: 'opportunity.admin@alkem.io',
-    id: '',
-    displayName: '',
-    profileId: '',
-    nameID: '',
-  },
-  {
-    email: 'opportunity.member@alkem.io',
-    id: '',
-    displayName: '',
-    profileId: '',
-    nameID: '',
-  },
-  {
-    email: 'qa.user@alkem.io',
-    id: '',
-    displayName: '',
-    profileId: '',
-    nameID: '',
-  },
-  {
-    email: 'notifications@alkem.io',
-    id: '',
-    displayName: '',
-    profileId: '',
-    nameID: '',
-  },
-  {
-    email: 'non.space@alkem.io',
-    id: '',
-    displayName: '',
-    profileId: '',
-    nameID: '',
-  },
-  {
-    email: 'beta.tester@alkem.io',
-    id: '',
-    displayName: '',
-    profileId: '',
-    nameID: '',
-  },
+export const usersSetEmail: UserData[] = [
```

Comment:
transform to an object, since tuples don't give enough information when types are concerned.
also add `as const` at the end of the object declaration. That will make `usersSetEmail` read only.